package.json

```diff
@@ -22,9 +22,11 @@
     "url": "https://github.com/karlhorky/react-malarkey/issues"
   },
   "homepage": "https://github.com/karlhorky/react-malarkey#readme",
+  "dependencies": {
+    "malarkey": "^1.3.3"
```

Comment:
Should this really be a `dependency`? I had it first as a dependency, but if an author is already using `malarkey` in their project, then this will force them to have two versions. `peerDependencies` leaves them a bit more flexibility to specify what they want and need.